### DISCUSSION TITLE
post upgrade commands with GitHub bot

### DISCUSSION THREAD

**Discussion by eps1lon** [renovatebot/renovate#10738] [Request Help] [answered]
We'd like to deduplicate packages in a npm monorepo after renovate bumps dependencies. It seems like the configuration has an option (`postUpgradeTasks`) but it's not clear how to get this to work with the GitHub app. It refers to `allowedPostUpgradeCommands` which is admin-only but  it's unclear where we would configure this in the GitHub app.

**Comment by viceice** [comment, marked as answer]
This is not possible with hosted app, only on self-hosted renovate.

Maybe `postUpdateOptions` works for you.

https://docs.renovatebot.com/configuration-options/#postupdateoptions
